pages/1.11/installing/ent/upgrading/index.md

```diff
@@ -15,7 +15,8 @@ If this upgrade is performed on a supported OS with all prerequisites fulfilled,
 **Important:**
 
 - Review the [release notes](/1.11/release-notes/) before upgrading DC/OS.
-- There are new options in the `config.yaml` file which must be declared prior to upgrading. Even if you have previously installed DC/OS successfully with your `config.yaml` file, the file will require new additions to function with DC/OS 1.11. Pay specific attention to the `fault_domain_enabled` and `enable_ipv6`. Please review the sample file [here](/latest/installing/ent/custom/advanced/#create-a-configuration-file)
+- Due to cluster configuration issues that caused overlay networks to fail in DC/OS 1.11.0, 1.11.1, and 1.11.2, we currently recommend setting `enable_ipv6` to `false` in `config.yaml` when upgrading _or_ configuring a new cluster. If you have already upgraded to DC/OS 1.11.x without configuring `enable_ipv6` or if `config.yaml` file is set to `true` then do not add new nodes until DC/OS 1.11.3 has been released. You can find additional information and a more robust remediation procedure in our latest critical product advisory.
```

Comment:
Should we change the first sentence to the following?
"Due to a cluster configuration issue with overlay networks, we currently recommend setting enable_ipv6 to false in config.yaml when upgrading or configuring a new cluster."
The phrase "issues that caused overlay networks to fail" seems negative and alarming. But if it's important to emphasize the severity of the issue so customers don't miss it, we can keep the sentence as is. @jenafermiller What's your preference?

Reply:
I like Pavi's suggestion.  It's serious but I don't want to alarm.  Let's make that change. 👍 I will have a product advisory link to add in a few minutes.

Reply:
SGTM. Please share the link here so we can track our changes/inputs in one place.